### ISSUE TITLE
Feature request, show appointment calendar

### ISSUE THREAD

**Issue by dannymertens** [atomic7777/atomic_calendar#72]
Hi,

I want to show my current day as a small snippet on my homescreen, works great but what I'm missing is an indication of the appointment calendar ( the calendar entity of the appointment)

![image](https://user-images.githubusercontent.com/46963011/78422726-bfccd980-7661-11ea-9f1f-6297d2b361e5.png)

**Comment by marksie1988** [comment]
Hey 

Do you mean that you want it to show the name of the calendar that the appointment is from? 

E.g. calendar.family

**Comment by dannymertens** [comment]
Yes, we show or family calendar and individual calendars on a dashboard and sometimes we have two appointments with the same name on a different time slot and this way we can see for who the appointment is ;-)

Thanks!

Kind regards,
Met vriendelijke groet,

Danny Mertens

**Comment by marksie1988** [comment]
@dannymertens please checkout [atomic-calendar-revive](https://github.com/marksie1988/atomic-calendar-revive) which has this feature and is the actively maintained version
